app/views/patients/_patients.html.erb

```diff
@@ -128,7 +128,7 @@
 
     function updateAssignedUserOptions() {
       // get a sorted, unique list of assigned users based on the scope
-      const assignedUsers = table.data()
+      const assigned_users = table.data()
                                  .filter(function (row) {
```

Comment:
small thing, but can we align the function calls below?

Reply:
done